src/position.cpp

```diff
@@ -599,8 +599,8 @@ bool Position::pseudo_legal(const Move m) const {
             if (!(between_bb(square<KING>(us), lsb(checkers())) & to))
                 return false;
         }
-        // In case of king moves under check we have to remove the king so as to catch
-        // invalid moves like b1a1 when opposite queen is on c1.
+        // For king moves under check, we temporarily remove the king to detect
```

Comment:
The word "temporarily" sounds incongruous because bitboard is not actually modified here.

Reply:
Removed.